src/01.UI-Elements/ListItem.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+ListItem.propTypes = {
+  titleListItem: PropTypes.string.isRequired,
+  isCheckmarked: PropTypes.bool,
+}
+
+export default function ListItem({ titleListItem, isCheckmarked }) {
+  return (
+    <ElementStyled>
```

Comment:
I would change the name to make it more understandable.

Reply:
Done.